requirements.txt

```diff
@@ -23,15 +23,15 @@ beautifulsoup4==4.3.2
 django-recaptcha==1.0.4
 
 # Api dependencies
-djangorestframework==3.1.1
+djangorestframework==3.3.2
 djangorestframework-xml==1.0.1
-django-filter==0.9.2
-django-oauth-toolkit==0.9.0
-drf-extensions==0.2.7
-django-rest-swagger==0.2.9
-django-cors-headers==1.0.0
+django-filter==0.12
+django-oauth-toolkit==0.10.0
+drf-extensions==0.2.8
+django-rest-swagger==0.3.4
+django-cors-middleware==1.2.0
 dry-rest-permissions==0.1.6
 
 # Zep 12 dependency
-django-uuslug==1.0.3
+django-uuslug==1.1.6
```

Comment:
A tester fortement, cette version opère des changements sur l'algo de slug.
si ça marche pas rollback 1.1.5
Si ça marche up to 1.1.7